k8s-mixin/defaults.libsonnet

```diff
@@ -233,7 +233,8 @@
         local serviceAccount = $.core.v1.serviceAccount,
 
         service_account:
-          serviceAccount.new(name),
+          serviceAccount.new(name) +
+          serviceAccount.metadata.withNamespace($._config.namespace),
```

Comment:
is `$._config.namespace` an expectation of this `k8s-mixin`? It isn't an expectation of `k8s-alpha`, is it?

Reply:
yeah, that would be part of splitting out the utils into a separate util lib that needs its own config